### ISSUE TITLE
FARO lsproj fails to load in CC 2.11 alpha 10/6/19

### ISSUE THREAD

**Issue by onitreb** [CloudCompare/CloudCompare#967]
Install cc 2.11 alpha 10/6/19 w/ the FARO sdk option

Try to load a FARO .lsproj file

Result:
error in CC:
[12:46:25] [Faro] Failed to connect to FARO LS runtime: Cannot instantiate FARO Open interface

Tried with various lsproj files, new and old. All produce the same error.

I have various FARO LS versions installed:
![image](https://user-images.githubusercontent.com/2154479/66428422-d81dad80-e9ca-11e9-9738-5f015b827970.png)

**Comment by nh2** [comment]
I see this error too in CloudCompare v2.13.alpha

**Comment by dgirardeau** [comment]
Yes, this issue is related to the FARO SDK and not to the version of CC sadly... 

CloudCompare v2.13.alpha requires Faro SDK 1.1.800.7 (it should be silently installed by the installer when checking the right option). Isn't it the case on your side?

**Comment by nh2** [comment]
@dgirardeau Yes, the CloudCompare v2.13.alpha Windows installer installed the Faro SDK for me, and i can see the plugin in CloudCompare's plugin list, but I still get the `Cannot instantiate FARO Open interface` error message when dragging a `.fls` in.

**Comment by dgirardeau** [comment]
I doubled check on my side and everything works. So my only bet is that it's either a Faro LS SDK installation issue, or an issue with the data (maybe it requires a more recent version of Faro LS SDK). I'll try to upgrade the LS SDK shipped with CC.

**Comment by dgirardeau** [comment]
Oh, I see that the LS SDK is not supported anymore by Faro :(
